cli.php

```diff
@@ -11,6 +11,10 @@ class WPSuperCache_Command extends WP_CLI_Command {
 	 * @synopsis [--post_id=<post-id>] [--permalink=<permalink>]
 	 */
 	function flush( $args = array(), $assoc_args = array() ) {
+		require_once( WPCACHEHOME . '/wp-cache-phase1.php' );
+		global $WPSC_HTTP_HOST;
+		$WPSC_HTTP_HOST = parse_url(get_site_url())['host'];
```

Comment:
I see but I still don't get why it's better to manually set `url` and `path` instead of automatically deducing them from the relevant WordPress installation. Worse yet, I bet many users are unaware of these variables which causes problems like in my case. What's your take on this, @danielbachhuber?